tests/fixtures/Payments/can_get_payment_submission.yaml

```diff
@@ -0,0 +1,13 @@
+-
+    request:
+        method: GET
+        url: 'https://devapi.currencycloud.com/v2/payments/48e707c9-43e3-4b07-a1d1-bee38f9c95a1/submission'
+        headers:
+            X-Auth-Token: e5070d4a16c5ffe4ed9fb268a2a716be
+    response:
+        status: 200
+        headers:
+            Date: 'Wed, 31 Oct 2018 11:57:56 GMT'
+            Content-Type: application/json
+            X-Request-Id: 7f0aa045-7175-4276-8d4d-4fd9d5e67a98
+        body: '{"status":null,"mt103":null,"submission_ref":null        }'
```

Comment:
A successful mt103 submission request will return a json response like the below example. Test must reflect this.

{
  "mt103": "{1:F01TCCLGB20AXXX0090000004}{2:I103BARCGB22XXXXN}{4: :20:20160617-ZSYWVY :23B:CRED :32A:160617GBP3000,0 :33B:GBP3000,0 :50K:/150618-00026 PCOMAPNY address New-York Province 555222 GB :53B:/20060513071472 :57C://SC200605 :59:/200605000 First Name Last Name e03036bf6c325dd12c58 London GB :70:test reference Test reason Payment group: 0160617-ZSYWVY :71A:SHA -}",
  "status": "pending",
  "submission_ref": "MXGGYAGJULIIQKDV"
}

Reply:
@EAddario I added the response in tests.